lib/datasource/terraform-module/index.ts

```diff
@@ -127,16 +81,63 @@ export async function getReleases({
     if (latestVersion) {
       latestVersion.releaseTimestamp = res.published_at;
     }
+
     logger.trace({ dep }, 'dep');
-    const cacheMinutes = 30;
-    await packageCache.set(cacheNamespace, pkgUrl, dep, cacheMinutes);
     return dep;
-  } catch (err) {
+  }
+
+  @cache({
+    namespace: 'terraform-service-discovery',
+    key: (registryUrl: string) => `${registryUrl}/.well-known/terraform.json`,
```

Comment:
What if `registryUrl`  has trailing slash? Can that happen?

Reply:
Good point. This is copied (unmodified) from the previous implementation, but it doesn't mean there wasn't a bug there too. We have a helper method for this, right?